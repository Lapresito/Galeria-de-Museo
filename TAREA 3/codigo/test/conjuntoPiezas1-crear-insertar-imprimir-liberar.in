crearConjuntoPiezas 2
imprimirConjuntoPiezas
liberarConjuntoPiezas
crearConjuntoPiezas 2
insertarConjuntoPiezas 1
insertarConjuntoPiezas 2
insertarConjuntoPiezas 1
imprimirConjuntoPiezas
liberarConjuntoPiezas
Fin