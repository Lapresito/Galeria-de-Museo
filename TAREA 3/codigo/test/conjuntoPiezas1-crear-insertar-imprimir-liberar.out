Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> El conjunto piezas fue creado en forma correcta.
2> Conjunto de piezas:

3> Conjunto de piezas liberado exitosamente.
4> El conjunto piezas fue creado en forma correcta.
5> Se ejecuto 'insertarTConjuntoPiezas' con pieza de id 1 exitosamente.
6> Se ejecuto 'insertarTConjuntoPiezas' con pieza de id 2 exitosamente.
7> Se ejecuto 'insertarTConjuntoPiezas' con pieza de id 1 exitosamente.
8> Conjunto de piezas:
1 
9> Conjunto de piezas liberado exitosamente.
10> Fin.
